ion/binarywriter.go

```diff
@@ -492,17 +492,19 @@ func (w *binaryWriter) beginValue(api string) error {
 	}
 
 	if len(as) > 0 {
-		ids := make([]uint64, len(as))
+		var ids []uint64
 		idlen := uint64(0)
 
-		for i, a := range as {
-			id, err := w.resolve(api, a)
-			if err != nil {
-				return err
-			}
+		for _, a := range as {
+			if a.Text != nil {
```

Comment:
Is silently dropping annotations with `Text: nil` the correct behavior here? Feels like we should either be writing out a.LocalSID or failing?

Reply:
Addressed.